index.js

```diff
@@ -74,22 +74,30 @@ function normalizeManifestPaths (tokensByFile, rootDir) {
   return output;
 }
 
+function dedupeSources (sources) {
+  var foundHashes = {}
+  Object.keys(sources).forEach(function (key) {
+    var hash = stringHash(sources[key]);
+    if (foundHashes[hash]) {
+      delete sources[key];
```

Comment:
Minor suggestion: This is silly, but using `delete` forces V8 to use un-optimized objects, resulting in a speed loss. It's a minor thing that we might not even notice, but an alternative way to write this might be:


```js
function dedeupSources (loader) {
var dedupedSources = {}
loader.sources = Object.keys(loader.sources).reduce(function (sources, key) {
  var hash = stringHash(loader.sources[key])
  if (dedeupedSources[hash]) {
    return sources
  }
  else {
   dedupedSources[hash] = true
   sources[key] = loader.sources[key]
   return sources
  }
})
```

It's a small optimization, but might matter in large code bases.